operators/multiclusterobservability/api/shared/multiclusterobservability_shared.go

```diff
@@ -15,8 +15,8 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-// URL is kubebuilder type that validates the containing string is an URL.
-// +kubebuilder:validation:Pattern=`^https?:\/\/`
+// URL is kubebuilder type that validates the containing string is an HTTPS URL.
+// +kubebuilder:validation:Pattern=`^https:\/\/`
```

Comment:
is there any risk here of this breaking anyone?

Reply:
No, because the creation of the additional config for cluster monitoring has been always hardcoded for HTTPS here: https://github.com/stolostron/multicluster-observability-operator/blob/21c0fdac879c3993a6ff1c1a7708c6897a2e4926/operators/endpointmetrics/controllers/observabilityendpoint/ocp_monitoring_config.go#L258